lib/D2L.Extensibility.AuthSdk.IntegrationTests/Helpers/ConfigHelper.cs

```diff
@@ -1,25 +1,22 @@
-﻿using System;
-using System.Configuration;
-
-namespace D2L.Extensibility.AuthSdk.IntegrationTests.Helpers {
-    internal static class ConfigHelper {
-
-        internal static string AppId { get; private set; }
-        internal static string AppKey { get; private set; }
-        internal static string UserId { get; private set; }
-        internal static string UserKey { get; private set; }
-        internal static string Scheme { get; private set; }
-        internal static string Host { get; private set; }
-        internal static int Port { get; private set; }
-
-        static ConfigHelper() {
-            AppId = ConfigurationManager.AppSettings["appId"];
-            AppKey = ConfigurationManager.AppSettings["appKey"];
-            UserId = ConfigurationManager.AppSettings["userId"];
-            UserKey = ConfigurationManager.AppSettings["userKey"];
-            Scheme = ConfigurationManager.AppSettings["scheme"];
-            Host = ConfigurationManager.AppSettings["host"];
-            Port = Int32.Parse( ConfigurationManager.AppSettings["port"] );
-        }
-    }
+﻿namespace D2L.Extensibility.AuthSdk.IntegrationTests.Helpers {
+	internal static class ConfigHelper {
+
+		internal static string AppId { get; private set; }
+		internal static string AppKey { get; private set; }
+		internal static string UserId { get; private set; }
+		internal static string UserKey { get; private set; }
+		internal static string Scheme { get; private set; }
+		internal static string Host { get; private set; }
+		internal static int Port { get; private set; }
+
+		static ConfigHelper() {
+			AppId = "G9nUpvbZQyiPrk3um2YAkQ";
+			AppKey = "ybZu7fm_JKJTFwKEHfoZ7Q";
+			UserId = "SomeUserId";
+			UserKey = "SomeUserKey";
+			Scheme = "https";
+			Host = "devcop.brightspacedemo.com";
+			Port = 443;

```

Comment:
Inlined from the config file (`ConfigurationManager` doesn't exist in .NET core)
So this is ugly but

(1) it probably never ran right off the bat because those are not a valid UserId/UserKey
(2) devcop.brightspace.com is an NXDOMAIN :P